daemon/container_unix.go

```diff
@@ -948,7 +948,8 @@ func (container *Container) connectToNetwork(idOrName string, updateSettings boo
 
 	ep, err := container.getEndpointInNetwork(n)
 	if err == nil {
-		return fmt.Errorf("container already connected to network %s", idOrName)
```

Comment:
Good point, I did not take multi-host into consideration there (although check on conflicting IP-address would probably still be a solution there)

I'll ask others for an opinion, but feel free to close if this is the wrong approach; it just felt it being redundant from a UX perspective